composer.json

```diff
@@ -72,9 +73,11 @@
         "ci:ts:lint": ".Build/vendor/bin/typoscript-lint -c Configuration/TsLint.yml --ansi -n --fail-on-warnings -vvv Configuration/TypoScript",
         "ci:tests:unit": ".Build/vendor/bin/phpunit -c .Build/vendor/nimut/testing-framework/res/Configuration/UnitTests.xml Tests/Unit",
         "ci:tests:functional": "find 'Tests/Functional' -wholename '*Test.php' | parallel --gnu 'echo; echo \"Running functional test suite {}\"; .Build/vendor/bin/phpunit -c .Build/vendor/nimut/testing-framework/res/Configuration/FunctionalTests.xml {}';",
+        "ci:tests:security": ".Build/vendor/bin/security-checker security:check",
         "ci:tests": [
             "@ci:tests:unit",
-            "@ci:tests:functional"
+            "@ci:tests:functional",
+            "@ci:tests:security"
```

Comment:
Are those really tests in the meaning of "dynamic tests along the lines of unit tests, functional tests, acceptance tests"? If not, I'd opt for moving this from `ci:tests:security` to `ci:security`.